src/main/java/frc/robot/Subsystems/Vision/VisionConstants.java

```diff
@@ -0,0 +1,126 @@
+import edu.wpi.first.math.geometry.Translation3d;
+
+
+
+public class VisionConstants {
+    /**
+     * Max height deviation for vision targets (in meters)
+     */
+    public static final double MAX_HEIGHT_DEV = 0.1;
+    /**
+     * Max ambiguity for multi-tag targets (0 to 1)
+     */
+    public static final double MAX_MULTI_AMBIGUTY = 0.3;
+    /**
+     * Max ambiguity for single-tag targets (0 to 1)
+     */
+    public static final double MAX_SINGLE_AMBIGUTY = 0.1;
+
+
+    // enum with all the camera constants
+    enum CamerasConstants {
+        // Define the camera constants for the front camera
+        FRONT_CAMERA(
+            PhotonPoseEstimator.PoseStrategy.MULTI_TAG_PNP_ON_COPROCESSOR,
+            PhotonPoseEstimator.PoseStrategy.LOWEST_AMBIGUITY,
+            "FrontCamera",
+            new Transform3d(
+                new Translation3d(0.2, 0.0, 0.5),
+                new Rotation3d(0, 0, 0)
+                
+            ),
+            0.1, // XY standard deviation factor
+            0.1, // Theta standard deviation factor
+             0.05,// Minimum XY standard deviation
+              Math.toRadians(5)// Minimum Theta standard deviation
+            
+        );
+        
+       
+        /**
+         * The main strategy for the camera
+         */
+        public final PhotonPoseEstimator.PoseStrategy MAIN_STRATEGY;
+        
+       /**
+        * The alternate strategy for the camera
+        */
+        public final PhotonPoseEstimator.PoseStrategy ALTERNATE_STRATEGY;
+
+        /**
+         * The name of the camera
+         */
+        public final String CAMERA_NAME;
+
+        /**
+         * The transform from the robot to the camera
+         */ 
+        public final Transform3d ROBOT_TO_CAMERA;
+
+        /**
+         * The standard deviation factor for XY
+         */
+        public final double XY_STD_DEV_FACTOR;
+
+        /**
+         * The standard deviation factor for Theta
+         */
+        public final double THETA_STD_DEV_FACTOR;
+        /**
+         * The minimum standard deviation for XY (in meters)
+         */
+        public final double MIN_XY_STD_DEV; 
+        /**
+         * The minimum standard deviation for Theta (in radians)
+         */
+        public final double MIN_THETA_STD_DEV; 
+
+        // Constructor for the camera constants
+        CamerasConstants(
+        PhotonPoseEstimator.PoseStrategy mainStrategy,
+        PhotonPoseEstimator.PoseStrategy alternateStrategy, 
+        String cameraName, 
+        Transform3d robotToCamera,
+        double xyStdFactor,
+        double thetaStdFactor,
+        double minXYStd,
+        double minThetaStd) {
+            
+            this.MAIN_STRATEGY = mainStrategy;
+
+            this.ALTERNATE_STRATEGY = alternateStrategy;
+
+            this.CAMERA_NAME = cameraName;
+
+            this.ROBOT_TO_CAMERA = robotToCamera;
+
+            //TODO: determaine unit for standard deviation factor
```

Comment:
All TODO comments should contain owner as well:
```suggestion
            // TODO(owner): determine unit for standard deviation factor
```